"password auth failed" in controller functional tests

Intermittently [this kind of stack trace](http://ci.deis.io/job/pull-request/40/console) in `make -C controller/ test-functional` has stopped a pull-request job in CI:

```
=== RUN TestController
--- Running deis/test-etcd at 10.0.1.15:54994
Error response from daemon: No such container: deis-etcd-jenkins-pull-request-40
[etcd] Sep 24 17:19:02.419 WARNING   | Using the directory 73424be92b08.etcd as the etcd curation directory because a directory was not specified. 
[etcd] Sep 24 17:19:02.420 INFO      | 73424be92b08 is starting a new cluster
[etcd] Sep 24 17:19:02.422 INFO      | etcd server [name 73424be92b08, listen on :54994, advertised url http://10.0.1.15:54994]
[etcd] Sep 24 17:19:02.422 INFO      | peer server [name 73424be92b08, listen on :7001, advertised url http://127.0.0.1:7001]
[etcd] Sep 24 17:19:02.422 INFO      | 73424be92b08 starting in peer mode
[etcd] Sep 24 17:19:02.422 INFO      | 73424be92b08: state changed from 'initialized' to 'follower'.
[etcd] Sep 24 17:19:02.422 INFO      | 73424be92b08: state changed from 'follower' to 'leader'.
[etcd] Sep 24 17:19:02.422 INFO      | 73424be92b08: leader changed from '' to '73424be92b08'.
Etcd client initialized
--- Publish etcd keys and values
*** Running /etc/rc.local...
*** Booting runit daemon...
*** Runit started as PID 8
POSTGRES_USER=deis
POSTGRES_PASS=deis
POSTGRES_DATA_DIR=/data
POSTGRES_DB=deis
Initializing PostgreSQL at /data
Starting PostgreSQL...
Etcd client initialized
--- Publish etcd keys and values
--- Run deis/controller:jenkins-pull-request-40 at 10.0.1.15:56830
Error response from daemon: No such container: deis-controller-jenkins-pull-request-40
Traceback (most recent call last):
Syncing...
  File "./manage.py", line 13, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/__init__.py", line 399, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/__init__.py", line 392, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/base.py", line 242, in run_from_argv
    self.execute(*args, **options.__dict__)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/base.py", line 285, in execute
    output = self.handle(*args, **options)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/base.py", line 415, in handle
    return self.handle_noargs(**options)
  File "/usr/local/lib/python2.7/dist-packages/south/management/commands/syncdb.py", line 92, in handle_noargs
    syncdb.Command().execute(**options)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/base.py", line 285, in execute
    output = self.handle(*args, **options)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/base.py", line 415, in handle
    return self.handle_noargs(**options)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/commands/syncdb.py", line 57, in handle_noargs
    cursor = connection.cursor()
  File "/usr/local/lib/python2.7/dist-packages/django/db/backends/__init__.py", line 162, in cursor
    cursor = util.CursorWrapper(self._cursor(), self)
  File "/usr/local/lib/python2.7/dist-packages/django/db/backends/__init__.py", line 132, in _cursor
    self.ensure_connection()
  File "/usr/local/lib/python2.7/dist-packages/django/db/backends/__init__.py", line 127, in ensure_connection
    self.connect()
  File "/usr/local/lib/python2.7/dist-packages/django/db/utils.py", line 99, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/usr/local/lib/python2.7/dist-packages/django/db/backends/__init__.py", line 127, in ensure_connection
    self.connect()
  File "/usr/local/lib/python2.7/dist-packages/django/db/backends/__init__.py", line 115, in connect
    self.connection = self.get_new_connection(conn_params)
  File "/usr/local/lib/python2.7/dist-packages/django/db/backends/postgresql_psycopg2/base.py", line 115, in get_new_connection
    return Database.connect(**conn_params)
  File "/usr/local/lib/python2.7/dist-packages/psycopg2/__init__.py", line 164, in connect
    conn = _connect(dsn, connection_factory=connection_factory, async=async)
django.db.utils.OperationalError: FATAL:  password authentication failed for user "deis"
FATAL:  password authentication failed for user "deis"

*** Test killed: ran too long (10m0s).
FAIL    github.com/deis/deis/controller/tests   605.019s
make[1]: *** [test-functional] Error 1
```

The line `Error response from daemon: No such container: deis-controller-jenkins-pull-request-40` is **not** the error, just the output of a defensive `docker rm -f` that we are not clever enough to silence.

--- a/tests/mock/mock.go
+++ b/tests/mock/mock.go
@@ -33,7 +33,7 @@ func RunMockDatabase(t *testing.T, tag string, etcdPort string, dbPort string) {
 			dbImage)
 	}()
 	time.Sleep(1000 * time.Millisecond)
-	dockercli.PrintToStdout(t, stdout, stdoutPipe, "Starting")
+	dockercli.PrintToStdout(t, stdout, stdoutPipe, "Initialization complete.")
 	setkeys := []string{
 		"/deis/database/user",
 		"/deis/database/password",

